package-lock.json

```diff
@@ -11,9 +11,9 @@
       "dev": true
     },
     "@babel/runtime-corejs2": {
-      "version": "7.15.3",
-      "resolved": "https://registry.npmjs.org/@babel/runtime-corejs2/-/runtime-corejs2-7.15.3.tgz",
-      "integrity": "sha512-iG7ypZmrdoKP1ckFurS8z97TR+Bqd6KaDsLQ9DiC/Rdxmrvy1nsCDlgfLNKfalbg9sFWdmIdNf+Hg+19XysSFg==",
+      "version": "7.16.0",
+      "resolved": "https://registry.npmjs.org/@babel/runtime-corejs2/-/runtime-corejs2-7.16.0.tgz",
+      "integrity": "sha512-PICpXRloQQ7AtgN+8xYbOxaopQ1vMB9RV0tsVe5ikx5/6y7iO8tYQiyAf3hIYxwz8AohnSXBuXorBacFOs21xg==",
```

Comment:
This seems to be an artifact leftover from the npm update --force. Seems safest to amend this commit, delete the package lock and npm install to make sure this is accurate.

Reply:
what commit is this from? I can't tell from your comment

Reply:
done